akka-docs/src/main/paradox/futures.md

```diff
@@ -391,6 +391,19 @@ Java
 
 @@@ div { .group-java }
 
+## Retry
+
+`akka.pattern.retry` will retry a `Future` some number of times with a delay between each attempt.
```

Comment:
This is not how Java people would view this pattern I think, change this paragraph depending on java/scala too?

Reply:
Could you point me to an example of how to do it?

Reply:
Look for `@java[` in the doc sources